stl/inc/__msvc_sanitizer_annotate_container.hpp

```diff
@@ -15,7 +15,28 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma push_macro("new")
 #undef new
 
-#if !defined(_M_CEE_PURE) && !(defined(_DISABLE_STRING_ANNOTATION) && defined(_DISABLE_VECTOR_ANNOTATION))
+#ifdef _DISABLE_ASAN_ANNOTATIONS
+
+#ifdef _ENABLE_ASAN_ANNOTATIONS_ON_UNSUPPORTED_PLATFORMS
+#error \
+    "Invalid set of defines - one cannot define both _ENABLE_ASAN_ANNOTATIONS_ON_UNSUPPORTED_PLATFORMS and _DISABLE_ASAN_ANNOTATIONS at the same time"

```

Comment:
Mega-nit: Obviously one _can_ define both - one just did!
```suggestion
    "Invalid set of defines: _ENABLE_ASAN_ANNOTATIONS_ON_UNSUPPORTED_PLATFORMS and _DISABLE_ASAN_ANNOTATIONS are mutually exclusive"
```

Reply:
Maybe strike "Invalid set of defines: " as well, to get this under 120 columns.

Reply:
FYI, this is an indication of editor misconfiguration; we should all be using format-on-save to avoid failures like this.

Reply:
I believe this is clang-format output. Putting just:
```c++
#error "Invalid set of defines - one cannot define both _ENABLE_ASAN_ANNOTATIONS_ON_UNSUPPORTED_PLATFORMS and _DISABLE_ASAN_ANNOTATIONS at the same time"
```
in a file and clang-formatting produces the still overlong:
```c++
#error \
    "Invalid set of defines - one cannot define both _ENABLE_ASAN_ANNOTATIONS_ON_UNSUPPORTED_PLATFORMS and _DISABLE_ASAN_ANNOTATIONS at the same time"
```

Reply:
I'm fine with everything else, but I do not believe in passing non-strings to `#error` :P

Reply:
That's fine for `#pragma message`, but `#error` is more formal; it _hates_ contractions. =)